girder/utility/server.py

```diff
@@ -173,12 +173,6 @@ def loadRouteTable(reconcileRoutes=False):
     def reconcileRouteTable(routeTable):
         hasChanged = False
 
-        # GIRDER_ROUTE_ID is a special route, which can't be removed
```

Comment:
The docstring for `loadRouteTable` needs to be updated to reflect that it's no longer dealing with "no longer active" plugins.